marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/exceptions/CustomExceptionHandler.java

```diff
@@ -41,6 +41,7 @@
     @ExceptionHandler(FailedRequestException.class)
     protected ResponseEntity<JsonNode> handleFailedRequestExceptionRequest(
         FailedRequestException failedRequestException) {
+        failedRequestException.printStackTrace();
```

Comment:
These should be removed, as @akshaysonvane has a separate ticket for logging these via logger.error.

Reply:
Yeah. Meant to back those changes out before pushing.